docs/src/ci.md

```diff
@@ -415,7 +415,7 @@ Large test suites can take very long to execute. By executing a preliminary test
 This will give you a faster feedback loop and slightly lower CI consumption while working on Pull Requests.
 To detect test files affected by your changeset, `--only-changed` analyses your suites' dependency graph. This is a heuristic and might miss tests, so it's important that you always run the full test suite after the preliminary test run.
 
-```yml js title=".github/workflows/playwright.yml"
+```yml js title=".github/workflows/playwright.yml" {20-23}
```

Comment:
which lines are you trying to highlight? the ones which are highlighted right now seem not related. Maybe 24-27 instead?

Also the color contrast ratio seems off, make sure its AAA compliant using the accessibility insights web extension in light and dark mode: 

<img width="1079" alt="image" src="https://github.com/user-attachments/assets/50c3bcb1-9cab-408d-adfe-fc6e8a2a8b54">